### DISCUSSION TITLE
Конфиг для билайна

### DISCUSSION THREAD

**Discussion by mad1van** [dovecoteescapee/ByeDPIAndroid#116] [Q&A]
Есть кто на билайне из Москвы или Питера? Удалось ли подобрать рабочий конфиг для домашнего интернета?

**Comment by mad1van** [comment]
**Upd**. Кажется нашел работающий у меня на билайн, да еще и по вайфай:
https://github.com/dovecoteescapee/ByeDPIAndroid/discussions/64#discussioncomment-10394799
По крайней мере пока полет нормальный. Смотрю, если что через smartube. 
**Upd.** А нет. Стоило для теста перезагрузить приставку и уже не робит.

**Comment by stepashka61** [comment]
В Ростове-на-дону такой конфиг вполне работает на версии 1.0.2
DNS:                       8.8.8.8

Mode:                      VPN

Desync only HTTPS and TLS: On

Desync method:             fake 

Split position:            1 

Split TLS record at SNI:   On

**Reply by mad1van** [reply]
Пробовал не помогло.

**Reply by KomarovS** [reply]
Здрав будь, добрый человек. Перешёл на Билайн сегодня, открывает всё кроме видео потока. С этими настройками всё заработало как раньше. Точнее как до того, как дом.ру обновил блокировки и перестало работать вообще.

**Reply by Heqq** [reply]
Тоже перешёл на Билайн, с этими настройками все заработало. Спасибо.

**Comment by mad1van** [comment]
Вообщем, кажется, нашел рабочую схему.

1. версия программы 1.2.0 + smartube 23.24 beta
2. режим прокси
3. конфиг **ingref**:

> **-s 2 -M h,d -A torst -T 3 -f -1 -r 3+s -S**

Посмотрим, надолго ли хватит.

**Comment by Daemon26** [comment]
К сожалению у меня тоже откис, и ничего пока не помогает, оператор Beeline

**Comment by 0pana** [comment]
-Ku -a3 -An -Kt,h -d1 -s0+s -s3+s -s6+s -s9+s -s12+s -s15+s -s20+s -s30+s -An

**Reply by Daemon26** [reply]
к сожалению не запустился :(

**Reply by 0pana** [reply]
Несколько дней назад еще работал и на мобильном и на домашнем интернете (Билайн Питер).

А сейчас только разные конфиги для разных подключений работают в [byebyedpi](https://github.com/romanvht/ByeDPIAndroid?tab=readme-ov-file#byebyedpi-android)

для домашнего инета и любого ютуба (и через домашний и через мобильный) работает такой конфиг
`-Ku -a3 -An -Kt,h -s0 -o1 -Ar -o1 -At -f-1 -r1+s -As -An`
а для мобильного без ютуба такой
`-n otus.ru -n google.com -d1:3 -f-1 -a2`

В byebyedpi можно сохранять избранные конфиги и переключаться между ними по необходимости
![Screenshot_20250701-225757_ByeByeDPI](https://github.com/user-attachments/assets/eaf38c81-bea3-439f-9b00-220eb13883f0)
